src/pg/brc20/brc20-pg-store.ts

```diff
@@ -156,6 +157,11 @@ export class Brc20PgStore extends BasePgStoreModule {
                     trans_balance = trans_balance - (SELECT amount FROM validated_transfer)
                   WHERE brc20_deploy_id = (SELECT brc20_deploy_id FROM validated_transfer)
                     AND address = (SELECT from_address FROM validated_transfer)
+                ),
+                address_event_type_count_increase AS (
+                  INSERT INTO brc20_counts_by_address_event_type (address, transfer_send)
+                  (SELECT from_address, 1 FROM validated_transfer)
```

Comment:
Does `1` need the `FROM` or does this ensure the validated_transfer worked?

Reply:
Exactly. If `validated_transfer` does not yield results then this INSERT does not happen.